src/data.ts

```diff
@@ -2,9 +2,10 @@ import { createClient } from '@supabase/supabase-js'
 import type { Provider } from '@supabase/supabase-js'
 import type { Setter } from './store'
 
-export interface Score {
+export interface ScoreInterface {
```

Comment:
I don't see why we add the word Interface here. It should either be `Score` or `IScore` and prefer the former.

Reply:
Score is interfering with the `Score`(constant) from supabase in finished.tsx

Reply:
Ah sorry, I renamed Item to Score before we started trying to use typescript. IScore is fine